src/types/wbgetentities.ts

```diff
@@ -0,0 +1,13 @@
+import type { UrlResultFormat } from './options.js'
+
+export type wbgetentities = {
```

Comment:
I think the type should be named `WbGetEntities` for easier readability?

Reply:
sure, what do you think @maxlath ?

Reply:
`WbGetEntities` sounds good!